io.catenax.week_based_capacity_group/2.0.0/WeekBasedCapacityGroup.ttl

```diff
@@ -0,0 +1,318 @@
+   samm:preferredName "Week-Based Capacity Group"@en ;
+   samm:description "An entity made up of the weekly actual and maximum capacities for a specific customer in a given time period. A capacity group is used to compare demand and capacity data for one or several similar bundled materials in a defined unit of measure."@en ;
+   samm:properties ( :capacityGroupId :name [ samm:property :supplierLocations; samm:optional true ] :customer :supplier [ samm:property :unitOfMeasure; samm:optional true ] [ samm:property :linkedDemandSeries; samm:optional true ] [ samm:property :capacities; samm:optional true ] :changedAt [ samm:property :linkedCapacityGroups; samm:optional true ] :unitOfMeasureIsOmitted ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:capacityGroupId a samm:Property ;
+   samm:preferredName "Capacity Group ID"@en ;
+   samm:description "The Capacity Group ID uniquely identifies the capacity group within the business relationship between a supplier and its customer."@en ;
+   samm:characteristic ext-uuid:UuidV4Trait ;
+   samm:exampleValue "0157ba42-d2a8-4e28-8565-7b07830c1110" .
+
+:name a samm:Property ;
+   samm:preferredName "Capacity Group Name"@en ;
+   samm:description "Name of the capacity group."@en ;
+   samm:characteristic :CapacityGroupName ;
+   samm:exampleValue "Spark Plugs on drilling machine for car model XYZ" .
+
+:supplierLocations a samm:Property ;
+   samm:preferredName "Supplier Locations"@en ;
+   samm:description "Set of BPNS indicating the supplier sites that will be fulfilling the demands associated with this capacity group."@en ;
+   samm:characteristic :SupplierLocationsCharacteristic ;
+   samm:exampleValue "BPNS8888888888XX" .
+
+:customer a samm:Property ;
+   samm:preferredName "Customer"@en ;
+   samm:description "The Business Partner Number (BPNL) of the party requesting materials from a supplier."@en ;
+   samm:characteristic ext-number:BpnlTrait ;
+   samm:exampleValue "BPNL8888888888XX" .
+
+:supplier a samm:Property ;
+   samm:preferredName "Supplier"@en ;
+   samm:description "The Business Partner Number (BPNL) of the party providing materials to a customer."@en ;
+   samm:characteristic ext-number:BpnlTrait ;
+   samm:exampleValue "BPNL6666666666YY" .
+
+:unitOfMeasure a samm:Property ;
+   samm:preferredName "Unit of Measure"@en ;
+   samm:description "Unit of measurement for capacity quantities."@en ;
+   samm:characteristic ext-quantity:ItemUnitEnumeration ;
+   samm:exampleValue "unit:piece"^^samm:curie .
+
+:linkedDemandSeries a samm:Property ;
+   samm:preferredName "Linked Demand Series"@en ;
+   samm:description "Set of demand series assigned to this capacity group."@en ;
+   samm:characteristic :LinkedDemandSeriesSet .
+
+:capacities a samm:Property ;
+   samm:preferredName "Capacities"@en ;
+   samm:description "A time series with week-based granularity along a given time period containing the capacity values."@en ;
+   samm:characteristic :CapacityTimeSeries .
```

Comment:
possible to provide example value here?

Reply:
example values are provided in the Capayity entity and it's properties